Automattic/wp-calypso
# Localize the Countries on the A4A signup form

Related to Automattic/i18n-issues#828

## Proposed Changes

Localize the country names on the signup form.

## Why are these changes being made?


Improve the signup experience.

## Testing Instructions



1. Run A4A dev environment locally with `yarn start-a8c-for-agencies`.
2. Open `/signup?locale=ja` (the locale param can be any Mag-16 language slug).
3. Verify that the countries list is in Japanese.

## Pre-merge Checklist
